aegis-web/yo/app/scripts/controllers/projectCtrl.js

```diff
@@ -292,6 +292,10 @@ angular.module('hopsWorksApp')
             self.goToDatasets = function () {
               self.goToUrl('datasets');
             };
+            
+            self.goToProject = function () {
```

Comment:
`goToProject` is misleading. I would rename that to `goToProjectActivities`.

Reply:
ok